src/vcsparser.core/MeasureAggregators/NumberOfAuthorsMeasureAggregator.cs

```diff
@@ -24,6 +24,20 @@ public int GetValueForNewMeasure(DailyCodeChurn dailyCodeChurn)
             return currentUniqueAuthorsPerFile[dailyCodeChurn.FileName].Count();
         }
 
+        public int GetValueForProjectMeasure()

```

Comment:
Can you use Dictionary<string, bool> instead of List<string> ? 

The reason is that Dictionary would be O(1) whilst List<string> would be O(n).